stdlib/streams/src/main/ballerina/streams/aggregators.bal

```diff
@@ -66,6 +67,24 @@ public type Sum object {
         Sum sumAggregator = new();
         return sumAggregator;
     }
+
+    public function saveState() returns map<any> {
+        return {
+            "iSum": self.iSum,
+            "fSum": self.fSum
+        };
+    }
+
+    public function restoreState(map<any> state) {
+        any? iSum = state["iSum"];
```

Comment:
Since `()` belongs to `any`
```suggestion
        any iSum = state["iSum"];
```

Applies to other similar usage too.

Reply:
fixed in ffc21df